packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -193,6 +194,12 @@ export function useRounds(variables: RoundsVariables) {
   };
 }
 
+function filterRoundsWithProjects(rounds: RoundOverview[]) {
+  return rounds.filter((round) => {
+    return round?.projects?.length !== undefined && round.projects?.length > 0;
```

Comment:
Just round.projects?.length is throwing prjects.length ccould be undefined error.

Reply:
Even with `round?.projects?.length` (with a ? after round)?

Reply:
`return round?.projects?.length !== undefined && round.projects?.length > 0` is actually simpler, since it returns `true` or `false` at a place where a boolean is expected. `round?.projects?.length?` is shorter but more complex, since it returns an integer or undefined at a place where a boolean is expected, and that is then converted to a boolean by JavaScript's implicit coercion. After being burned by implicit coercion too many times, I now always prefer explicit to implicit.